### PR TITLE
Chapter 13: User Microposts

### PR DESCRIPTION
**Pull request by damhung0113**
image

### CODE REVIEW

**Comment by damhung0113** [comment]
ready

File: app/models/micropost.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :sort_by_time, ->{order created_at: :desc}
+  scope :news_feed, ->(id){where("user_id = ?", id)}
```
Comment (hungct-1121) [inline app/models/micropost.rb]:
```suggestion
  scope :news_feed, ->(id){where user_id: id}
```

**Comment by damhung0113** [comment]
ready

File: app/models/micropost.rb (hunk 1 above)
Comment (damhung0113) [reply app/models/micropost.rb]: :cry:

**Comment by hungct-1121** [comment]
merged

**Pull request merged**